packages/design-picker/src/__tests__/integration.test.tsx

```diff
@@ -1,6 +1,47 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { isEnabled } from '@automattic/calypso-config';
+
+/**
+ * Internal dependencies
+ */
+import DesignPicker from '../components';
+import { availableDesigns } from '../utils';
+import type { Design } from '../types';
+
+jest.mock( `@automattic/calypso-config`, () => ( {
+	isEnabled: jest.fn().mockImplementation( ( feature: string ) => {
+		switch ( feature ) {
+			case `gutenboarding/landscape-preview`:
+				return false;
+			case `gutenboarding/mshot-preview`:
+				return false;
+		}
```

Comment:
`gutenboarding/landscape-preview` is used to toggle between landscape and portrait mode.
`gutenboarding/mshot-preview` is used to toggle mshot previews.